novawallet/Common/Services/ExtrinsicService/Substrate/ExtrinsicServiceFactory.swift

```diff
@@ -117,7 +117,10 @@ extension ExtrinsicServiceFactory: ExtrinsicServiceFactoryProtocol {
 
         let feeEstimationRegistry = ExtrinsicFeeEstimationRegistry(
             chain: chain,
-            estimatingWrapperFactory: feeEstimatingWrapperFactory
+            estimatingWrapperFactory: feeEstimatingWrapperFactory,
+            chainRegistry: ChainRegistryFacade.sharedRegistry,
+            userStorageFacade: userStorageFacade,
+            substrateStorageFacade: SubstrateDataStorageFacade.shared
```

Comment:
We already have substrate storage facade in the init. We just need to store it and pass here